apps/transport/lib/datagouvfr/client/oauth.ex

```diff
@@ -47,6 +49,10 @@ defmodule Datagouvfr.Client.OAuth do
     url = process_url(path)
     opts = Keyword.put_new(opts, :timeout, 15_000)
     opts = Keyword.put_new(opts, :follow_redirect, true)
+    Logger.debug fn -> "Request to: #{path}" end
```

Comment:
do you want to keep those logs ?

Reply:
yep this is useful when you develop to know exactly what happened